### PR TITLE
WINDUP-3923: Document Rules Override enhancement

### PR DESCRIPTION
**Pull request by sbeskin-redhat**
Resolves: https://issues.redhat.com/browse/WINDUP-3923

MTR 1.2.0

Preview: https://deploy-preview-737--windup-documentation.netlify.app/docs/rules-development-guide/master/#overriding-a-rule

### CODE REVIEW

File: docs/topics/overriding-rules.adoc [hunk 1]
```diff
@@ -72,6 +74,8 @@ The following rule override example changes the `effort` of the `weblogic-02000`
 
 When you run {ProductShortName}, this rule overrides the original rule with the same rule ID. You can verify that the new rule was used by viewing the contents of the Rule Provider Executions Overview.
 
+In order for the override rules to run, `targetTechnology` within the override ruleset must match one of the targets selected by the user when the analysis is executed.
```
Comment (anarnold97) [inline docs/topics/overriding-rules.adoc:77]: I am not certain, but is the word 'executed' not problematic?

Comment (sbeskin-redhat) [reply docs/topics/overriding-rules.adoc]: This phrase should not be here at all.  i put it elsewhere and rephrased.

**Review by kpunwatk — APPROVED** [review]
LGTM :+1:

**Review by stillalearner — APPROVED** [review]
lgtm

File: docs/topics/overriding-rules.adoc [hunk 2]
```diff
@@ -29,6 +29,8 @@ Rules from the original ruleset that are not overridden by the new ruleset are r
 
 . Ensure that you keep the same rule and ruleset IDs. When you copy the original rule XML, this ensures that the IDs match.
 
+. Ensure that `targetTechnology` within the override ruleset matches one of the targets that you specified for running the analysis.
+
```
Comment (RichardHoch) [inline docs/topics/overriding-rules.adoc:33]: Suggestion: "...that the `targetTechnology` included in the override ruleset matches..." .

Comment (sbeskin-redhat) [reply docs/topics/overriding-rules.adoc:33]: @RichardHoch 
Yes, i would have certainly put 'the' there, had `targetTechnology` been a common noun. But the backticks make it sort of a proper noun - won't you agree? 
But how about I change it to this: "Ensure that the target technology in the override ruleset matches..."?

Comment (RichardHoch) [reply docs/topics/overriding-rules.adoc:33]: Your solution works. I don't think the backticks make the item a proper noun, but some proper nouns such "the Cabinet" take "the" anyway. So either go with "the `targetTechnology` .." or "the target technology," whichever you think is best.

**Pull request merged**